Source/Scene/Expression.js

```diff
@@ -52,6 +52,21 @@ define([
         setEvaluateFunction(this);
     }
 
+    function replaceVariables(expression) {
```

Comment:
OK, can you point me in the direction of the simpler way to implement this?

Also I can add an escape character.